test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import winnowing
+
+
+class TestWinnowing(unittest.TestCase):
+    # testing for R language
+    def test1(self):
+        check = winnowing.Winnowing(file1= "/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test6.r",file2="/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test7.r",lang="r")
```

Comment:
Add a setup function in the class and create relative file paths in the setup function.